### PR TITLE
update readmes and description

### PR DESCRIPTION
**Pull request by m7pr**
Related to https://github.com/insightsengineering/coredev-tasks/issues/462

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -19,8 +19,13 @@ Authors@R: c(
     person("Junlue", "Zhao", role = "aut"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Filter module for teal applications.
-License: Apache License 2.0 | file LICENSE
+Description: 
+    Module containing filter panel for 'teal' applications. Allows for interactive filtering of data stored in 
+    'data.frame' and 'MultiAssayExperiment' objects. Also displays filtered and unfiltered observation counts.
```
Comment (donyunardi) [inline DESCRIPTION:24]:
```suggestion
A filter panel module for 'teal' applications that allows interactive and reproducible data filtering.
```

I want to be careful to mention MAE in teal.slice. We have some discussion on moving this extension away from the core package.
Also, I want to mention about the reproducible part.
Let me know what you think.

**Review by donyunardi — CHANGES_REQUESTED** [review]
Thanks for working on this!
I have a few minor updates that I noticed.

File: vignettes/filter-panel-for-developers.Rmd [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "Filter panel for developers"
```
[the remark this answers is not in the document]
Comment (chlebowa) [reply vignettes/filter-panel-for-developers.Rmd:2]: Let's pick title case or sentence case and stick to it. Currently all titles are in sentence case, this correction is neither.

File: DESCRIPTION (hunk 1 above)
Comment (chlebowa) [reply DESCRIPTION:24]: It does handle MAE now so it should be mentioned, shouldn't it? When MAEs are moved to a different package, vignettes will be updated (in more places than one).

```
Module containing filter panel for 'teal' applications. Allows for interactive filtering of data stored in 'data.frame' and 'MultiAssayExperiment' objects nad returns reproducible code of the filtering operations. Also displays filtered and unfiltered observation counts.
```

File: vignettes/filter-panel-for-developers.Rmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (m7pr) [reply vignettes/filter-panel-for-developers.Rmd:2]: I'll stick to sentence case as it's the same in other vignettes

File: vignettes/teal-slice-classes.Rmd [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "teal.slice classes"
```
[the remark this answers is not in the document]
Comment (m7pr) [reply vignettes/teal-slice-classes.Rmd:2]: will leave it sentence case to remain consistent elswehere

**Comment by m7pr** [comment]
Hey @donyunardi I changed the name of the team in titles, but I left titles in sentence case. Thanks for the review!

**Comment by donyunardi** [comment]
Pushing very minor update to match vignette's title with `VignetteIndexEntry`.

**Review by donyunardi — APPROVED** [review]
LGTM!

**Pull request merged**